sdk/src/connections/network/buffer.proto

```diff
@@ -60,7 +70,7 @@ message ClientRequest
   repeated bytes func_bytes_param = 40;    // List of function parameters of type byte-array
   repeated string func_strings_param = 45; // List of function parameters of type string
   bool expect_reply = 50;                  // Whether a response with data is expected or not
-  DepthSensorFrameType frame_type = 60;    // Frame type information
+  repeated DepthSensorFrameType depthSensorFrameType = 60;    // Frame type information
```

Comment:
Why the need for 'repeated'?

Reply:
There is no need for repeated. Will be removed